CHANGELOG.md

```diff
@@ -1,3 +1,108 @@
+# 0.0.106 - 2022-03-30
+
+## API Updates
+ * Minimum supported rust version (MSRV) is now 1.41.1 (#1310).
+ * `PhantomRouteHints`, `FixedPenaltyScorer`, and `ScoringParameters` now
+   implement `Clone` (#1346).
+ * `DefaultRouter` and `find_route` take an additional random seed to improve
+   privacy by adding a random CLTV expiry offset to each path's final hop. This
+   helps obscure the intended recipient from adversarial intermediate hops
+   (#1286). The seed is  also used to randomize candidate paths during route
+   selection (#1359).
+ * `PeerManager::timer_tick_occurred` will disconnect an unresponsive peer after
+   four ticks, corresponding to 40 seconds with the new recommendation to call
+   the method every 10 seconds (#1348).
+ * The `lightning-block-sync` crate's `init::synchronize_listeners` method
+   interface has been relaxed to support multithreaded environments (#1349).
+ * `ChannelManager::create_inbound_payment_for_hash`'s documentation has been
+   corrected to remove the one-year restriction on `invoice_expiry_delta_secs`,
+   which is only applicable to the deprecated `create_inbound_payment_legacy`
+   and `create_inbound_payment_for_hash_legacy` methods (#1341).
+ * `ChannelManager` supports sending and receiving short channel id aliases in
+   the `funding_locked` message. These are used when forwarding payments and
+   constructing invoice route hints for improved privacy (#1311).
+ * `ChannelDetails` now has a `inbound_scid_alias` field and
+   `get_inbound_payment_scid` method (#1311).
+ * `ChannelManager` no longer fails a channel when receiving redundant
+   `funding_locked` messages since they may be used to update the channel's
+   short channel id alias (#1311).
+ * `Features` mutator methods now take `self` by reference instead of by value
+   (#1331).
+ * Invoice creation functions in `lightning-invoice` crate's `utils` module
+   filter invoice hints in order to limit the invoice size (#1325).
+ * Invoice creation functions in `lightning-invoice` crate's `utils` module
+   include versions that accept a description hash instead of only a description
+   (#1361).
+ * `RoutingMessageHandler::sync_routing_table` has been renamed `peer_connected`
+   (#1368).
+ * `MessageSendEvent::SendGossipTimestampFilter` has been added to indicate that
+   a `gossip_timestamp_filter` should be sent (#1368).
+ * `PeerManager` takes an optional `NetAddress` in `new_outbound_connection` and
+   `new_inbound_connection`, used to report back the remote address to the
+   connecting peer in the `init` message (#1326).
+ * `ChannelManager::accept_inbound_channel` now takes a `user_channel_id`, which
+   is given back in `Event::ChannelClosed::user_channel_id` (#1381).
```

Comment:
It is used in many more places. Maybe just mention that it now takes a user_channel_id which is equivalent to the one in outbound channels?